src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1775,6 +1775,30 @@
                         </UserData>
                     </template>
 
+                    <template tid="WaitStart_V1">
+                        <data name="ClrInstanceID" inType="win:UInt16" />
+                        <data name="LockID" inType="win:Pointer" />
+                        <data name="AssociatedObjectID" inType="win:Pointer" />
+                        <UserData>
+                            <Contention xmlns="myNs">
```

Comment:
The new event is under the keyword `contention (0x400)`. I think it makes sense to collect it whenever the contention events are collected

Reply:
I'd suggest adding a new keyword instead. The general rule of thumb is that if an event is likely to occur more than 1000 times/sec then we'd like to give it a unique keyword so that tracing tools can toggle it independently. Its a lot better to have users that want multiple events enable several keywords vs. have users that only want one kind of event forced to pay the performance overhead for noisy events they didn't need.

Reply:
PerfView has a way to enable/disable individual events under a keyword, not sure about dotnet-trace. Would it be beneficial (also for ease-of-use) to have one keyword for a variety of potentially spammy events? I'm not sure there are a lot of keywords available for unique things like this. There is already a very noisy event in the Threading keyword ([ThreadPoolWorkerThreadWait](https://learn.microsoft.com/en-us/dotnet/fundamentals/diagnostics/runtime-thread-events#threadpoolworkerthreadwait-event) that I've seen to affect perf characteristics of apps when enabled. Wonder if that also should be moved to a different keyword (or be diabled by default, and enabled via config, for compat reasons)?

Reply:
I moved it to a new keyword `WaitHandle`